File: examples/eApi v1.9/java/eAPI-v19-connector/src/main/java/cz/monetplus/mips/eapi/v19/connector/entity/responses/ApplepayInitParams.java
package cz.monetplus.mips.eapi.v19.connector.entity.responses;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import cz.monetplus.mips.eapi.v19.connector.entity.SignBase;
import lombok.*;

import java.util.List;

@EqualsAndHashCode(callSuper = false)
@Data
@AllArgsConstructor @NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
class ApplepayInitParams extends SignBase {
    private String countryCode;
    private List<String> supportedNetworks;
    private List<String> merchantCapabilities;

    @Override
    public String toSign() {
        StringBuilder sb = new StringBuilder();
        add(sb, countryCode);
        for (String network : supportedNetworks) add(sb, network);
        for (String capability : merchantCapabilities) add(sb, capability);
        deleteLast(sb);
        return sb.toString();
    }
}

File: examples/eApi v1.9/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/DataObjects/Response/ApplepayInitParams.cs
using System.Text;

namespace CsobGatewayClientExample.Communication.DataObjects.Response;

public class ApplepayInitParams : SignBase
{
    public string? CountryCode;
    public List<string>? SupportedNetworks { get; set; }
    public List<string>? MerchantCapabilities { get; set; }

    public override string ToSign()
    {
        var sb = new StringBuilder();
        Add(sb, CountryCode);
        if (SupportedNetworks != null)
            foreach (var value in SupportedNetworks)
                Add(sb, value);
        if (MerchantCapabilities != null)
            foreach (var value in MerchantCapabilities)
                Add(sb, value);
        DeleteLast(sb);
        return sb.ToString();
    }
}